src/Makefile.in

```diff
@@ -428,7 +430,7 @@ loader.o:	loader.c autoconfig.h jumbo.h arch.h stdint.h os.h os-autoconf.h misc.
 
 logger.o:	logger.c os.h os-autoconf.h autoconfig.h jumbo.h arch.h stdint.h misc.h params.h path.h memory.h status.h math.h options.h list.h loader.h formats.h getopt.h common.h config.h recovery.h unicode.h dynamic.h simd-intrinsics.h pseudo_intrinsics.h aligned.h simd-intrinsics-load-flags.h john-mpi.h cracker.h signals.h memdbg.h
 
-mask.o:	mask.c misc.h jumbo.h arch.h stdint.h autoconfig.h logger.h recovery.h loader.h params.h list.h formats.h os.h os-autoconf.h signals.h status.h math.h options.h getopt.h common.h memory.h config.h external.h compiler.h cracker.h john.h mask.h unicode.h encoding_data.h memdbg.h mask_ext.h opencl_mask.h
+mask.o:	mask.c misc.h jumbo.h arch.h stdint.h autoconfig.h logger.h recovery.h loader.h params.h list.h formats.h os.h os-autoconf.h signals.h status.h math.h options.h getopt.h common.h memory.h config.h external.h maskfile.h compiler.h cracker.h john.h mask.h unicode.h encoding_data.h memdbg.h mask_ext.h opencl_mask.h
```

Comment:
I mean between external.h and maskfile.h. But I guess is github playing with me

![image](https://cloud.githubusercontent.com/assets/1702923/20276118/1e0cf6e2-aa82-11e6-80bb-8500caadc141.png)